examples_utils/notebook_logging/gc_logger.py

```diff
@@ -540,9 +557,15 @@ def post_run_cell(cls, result):
                 cls._PAYLOAD["time_to_first_error_seconds"] = event_dict["logger_uptime_seconds"]
 
             event_dict["event_type"] = "error"
-            event_dict["error_trace"] = (
-                str(result.error_before_exec) if result.error_before_exec else str(result.error_in_exec)
-            )
+
+            # Get the stderr in the correct order
+            if result.error_before_exec is not None:
+                event_dict["error_trace"] = str(result.error_before_exec).replace("\n", "")
+            elif result.error_in_exec is not None:
+                event_dict["error_trace"] = str(result.error_in_exec).replace("\n", "")
```

Comment:
These two fields would never be needed together?

Reply:
I think so, I'm not 100% sure so I can check but I think I have seen cases where a cell ouptuts to stderr but can continue and end up terminating with more stderr. I'll try it out